drivers/slip/Kconfig

```diff
@@ -71,11 +71,10 @@ config	SLIP_STATISTICS
 
 config  SLIP_TAP
 	bool "Use TAP interface to host"
-	default n
+	default y
```

Comment:
At the moment SLIP requires TAP to be useful, as we only use SLIP to enable ethernet connection to host, so imho makes sense here.